internal/collector/otelcol.tmpl

```diff
@@ -1,15 +1,5 @@
 receivers:
-  {{- range .Receivers }}
-  {{- if eq .Type "otlp" }}
-  otlp:
-    protocols:
-      {{- if eq .Server.Type 0 }}
-      grpc:
-      {{- else }}
-      http:
-      {{- end }}
-        endpoint: "{{- .Server.Host -}}:{{- .Server.Port -}}"
-  {{- else if eq .Type "hostmetrics" }}
+{{- if .Receivers.HostMetrics }}
```

Comment:
as discussed, move host under metrics and expose collection_interval and initial_delay